### DISCUSSION TITLE
Hombre Python installation

### DISCUSSION THREAD

**Discussion by teamrocket77** [Homebrew/discussions#2904] [Everyday usage]
Hello all,
Is there a way to better manage python dependencies in home-brew. I wound up with Python@3.8, 3.9, 3.10 all due to dependencies and this was messing with my virtual environments

**Comment by teamrocket77** [comment]
I've been looking at additional information concerning that using pyenv with symlinks seems to be what I'm looking for additional recommendations still wanted

**Comment by SMillerDev** [comment]
Homebrew python does not affect your virtual environments. And the only way not to have python versions installed as dependencies is to avoid the software that has them as dependencies.

**Reply by carlocab** [reply]
Yup, this. The entire point of a virtual environment is that it shouldn't be affected by what else you've installed in your system.

Were your virtual environments created using a Homebrew-provided Python?

If not, then either your non-Homebrew Python installations are broken or there is a bug in Python's virtual environment module.

**Reply by teamrocket77** [reply]
Good point, I neglected to mention that I had an installation of conda top of of all of that maybe that was affecting it. I've since installed Pyenv and used that to globally control what's going on now.
